src/Commands/AbstractDiffVendorCommand.php

```
@@ -0,0 +1,370 @@
+                foreach ($unchanged as $f) {
+                    $fs->delete($f);
+                    $this->line('  ✖ deleted ' . $this->toRelativePath($f));
+                }
+            }
+        }
+    }
+
+    /**
+     * Strip all PHP comments from the content (always applied).
+     */
+    protected function stripComments(string $s): string
```

PHPMD (2.15.0)

Avoid variables with short names like $s. Configured minimum length is 3. (undefined)

(ShortVariable)